composer.json

```diff
@@ -31,6 +32,7 @@
         "symfony/console": "*",
         "symfony/flex": "*",
         "symfony/framework-bundle": "*",
+        "symfony/http-client": "*",
```

Comment:
Should we remove "php-http/guzzle6-adapter"?

Reply:
For now no! I only refactored one service for now!